content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -21,7 +21,7 @@ You must use a CNI plugin that is compatible with the
 
 ## Installation
 
-The kubelet has a single default network plugin, and a default network common to the entire cluster.The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
+The kubelet has a single default network plugin, and a default network common to the entire cluster. The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
```

Comment:
I'm (still) not sure that “The kubelet has a single default network plugin” remains true after the removal of dockershim.

Reply:
```suggestion
A CNI plugin is required to implement the [Kubernetes networking model](/docs/concepts/services-networking/#the-kubernetes-network-model). The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
```
I think @sftim is correct. With 1.24 and the removal of dockershim and kubenet then a CNI plugin is required. A technical review is required to confirm. 
Above is only a suggestion but will need a tech review.
/sig node